src/components/Facility/CentralNursingStation.tsx

```diff
@@ -41,6 +40,12 @@ interface Props {
 }
 
 export default function CentralNursingStation({ facilityId }: Props) {
+  const breakpointValues = {
+    default: 6,
+    largedisplay: 9,
+    fourK: 24,
+  };
+  const PER_PAGE_LIMIT = useBreakpoints(breakpointValues);
```

Comment:
1. use proper naming conventions
2. inline the breakpoint values so that type-aware autocomplete is available

```suggestion
  const perPageLimit = useBreakpoints({
    default: 6,
    "4xl": 9,
    "4k": 24,
  });
```